packages/ui-components/loader/cdn.js

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Delete this folder (`packages/ui-components/loader`) that you don't need

Reply:
This directory should be under the gitignore file.